bank2ynab/__main__.py

```diff
@@ -17,6 +17,6 @@
     else:
         b2y = Bank2Ynab(config)
         b2y.run()
-        api = YNAB_API(config)
-        if b2y.transaction_data:
-            api.run(b2y.transaction_data)
+        # api = YNAB_API(config) # DEBUG: disabled
+        """ if b2y.transaction_data: # DEBUG - disabled while testing
+            api.run(b2y.transaction_data) """
```

Comment:
1. try get configuration files (main & user-config [maybe payees mapping too])
    - Gracefully fail if there's no config file
2. for each section in the configuration file, create a bank class with the relevant parameters saved
3. create a list of bank classes
4. for each bank class in the list
    - find all matching files
    - read data into a dataframe from each file
    - attempt to parse & clean the data
    - validate data
    - write data to output csv file
5. output all transaction data to YNAB api